spec/requests/admin/ai_personas_controller_spec.rb

```diff
@@ -17,21 +17,40 @@
       )
     end
 
+    it "sideloads llms" do
+      get "/admin/plugins/discourse-ai/ai_personas.json"
```

Comment:
I thought the convention would be `/.../.../ai-personas.json`

Reply:
Yeah I agree URLs should have `-` not `_`, but this is probably annoying to change now.